facets.py

```diff
@@ -21,10 +31,25 @@ class FacetConstants(object):
     ORDER_SERIES_POSITION = 'series'
     ORDER_WORK_ID = 'work_id'
     ORDER_RANDOM = 'random'
+    ORDER_FACETS = [
+        ORDER_TITLE,
+        ORDER_AUTHOR,
+        ORDER_LAST_UPDATE,
+        ORDER_ADDED_TO_COLLECTION,
+        ORDER_SERIES_POSITION,
+        ORDER_WORK_ID,
+        ORDER_RANDOM,
+    ]
```

Comment:
Some of these facets only make sense in certain contexts and shouldn't be options in the user interface, which controls how lane feeds are presented to patrons. "random" might be fun, but I don't think there's any reason to include "last update" or "work id", and series position won't work at all.